### PR TITLE
wsdd: 0.7.0 -> 0.7.1

### PR DESCRIPTION
**Pull request by Izorkin**
###### Description of changes
Fixed samba-wsdd test.
Added manual.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/wsdd/default.nix [hunk 1]
```diff
@@ -19,10 +21,15 @@ stdenv.mkDerivation rec {
     # Increase timeout to socket urlopen
     # See https://github.com/christgau/wsdd/issues/80#issuecomment-76848906
     ./increase_timeout.patch
+    # Implicitly use currently running asyncio loop
+    # The patch is based on this commit:
+    # https://github.com/christgau/wsdd/commit/0aef2d73c01162d746414c25f1556ae510289cf4
+    ./implicitly_yuse_currently_running_asyncio_loop.patch
```
Comment (MinerSebas) [inline pkgs/servers/wsdd/default.nix:27]: Typo in filename, the actual patch file name would also need to be changed.
```suggestion
    ./implicitly_use_currently_running_asyncio_loop.patch
```

**Review by MinerSebas — APPROVED** [review]

File: pkgs/servers/wsdd/default.nix (hunk 1 above)
Comment (Izorkin) [reply pkgs/servers/wsdd/default.nix]: Fixed!

**Comment by Izorkin** [comment]
@ofborg eval

**Comment by Izorkin** [comment]
@GrahamcOfBorg eval

**Comment by Izorkin** [comment]
@GrahamcOfBorg test samba-wsdd

**Comment by Izorkin** [comment]
cc @SuperSandro2000

File: pkgs/servers/wsdd/default.nix [hunk 2]
```diff
@@ -19,10 +21,15 @@ stdenv.mkDerivation rec {
...
+    # Implicitly use currently running asyncio loop
+    # The patch is based on this commit:
+    # https://github.com/christgau/wsdd/commit/0aef2d73c01162d746414c25f1556ae510289cf4
+    ./implicitly_use_currently_running_asyncio_loop.patch
   ];
 
   installPhase = ''
-    install -Dm0755 src/wsdd.py $out/bin/wsdd
+    install -Dm0555 src/wsdd.py $out/bin/wsdd
+    install -Dm0444 man/wsdd.1 $man/share/man/man1/wsdd.1
```
Comment (SuperSandro2000) [inline pkgs/servers/wsdd/default.nix:32]:
```suggestion
    installManPage man/wsdd.1
```

Comment (Izorkin) [reply pkgs/servers/wsdd/default.nix]: Thanks, fixed.

**Comment by MinerSebas** [comment]
`wsdd` has released [`0.7.1` release](https://github.com/christgau/wsdd/releases/tag/v0.7.1) which includes the fix.

**Pull request merged**